DEVELOPING.md

```diff
@@ -31,7 +31,7 @@
 
 ## <a name="setup"></a> Setting up a New Development Environment
 
-1. Ensure you have Python (version `3.8`, `3.9` or `3.10`) and [`poetry`][poetry].
+1. Ensure you have Python (version `3.9` or `3.10`) and [`poetry`][poetry].
```

Comment:
Shouldn't the `3.11` be listed here as well?

Reply:
And 3.12